draft-ietf-ohai-ohttp.md

```diff
@@ -290,6 +299,22 @@ Oblivious Target Resource:
   logically handles only regular HTTP requests and responses and so might be
   ignorant of the use of oblivious HTTP to reach it.
 
+Relay Request:
+
+: An HTTP request from Client to Relay that contains an encapsulated request as its body.
```

Comment:
Let's use content instead of body here and subsequent defs

Reply:
```suggestion
: An HTTP request from Client to Relay that contains an encapsulated request as the content.
```

Reply:
@LPardue applied this series of fixes